agent/runner/actions/mysql_explain_action.go

```diff
@@ -258,3 +254,17 @@ func (a *mysqlExplainAction) explainTraditionalJSON(ctx context.Context, tx *sql
 	}
 	return jsonRows(columns, dataRows)
 }
+
+func checkExplainError(err error, schema string) error {
+	if err != nil {
+		if strings.Contains(err.Error(), errNoDatabaseSelectedCode) {
```

Comment:
Is there a way to avoid string comparison? Can't we leverage `errors.Is` or `errors.As` somehow? I believe it will make the code more maintainable.